typescript/src/Writer.ts

```diff
@@ -0,0 +1,77 @@
+export const LITTLE_ENDIAN = true;
+
+export default class Writer {
```

Comment:
When working on the bag2proto I felt like this should probably have a better name. Like DatatypeWriter or some such.